2019-07-01.md

```diff
@@ -0,0 +1,15 @@
+# Meta-Meetup 2019-07-01
+
+## Agenda
+
+- Upcoming meetup
+  - [Talk Selection](https://github.com/hamburg-js/proposals/issues)
+  - Location confirmation (Finanzcheck)
+  - Roles and Responsibilities
+- [FAQ review](https://github.com/hamburg-js/proposals/pull/11)
+- Lightning talks
+- NPM Roulette
+
```

Comment:
I would also like to talk about communication channels: It would be nice if we could find one or two persons who are responsible for our twitter account.